extensions/wikia/Search/classes/Test/IndexerTest.php

```diff
@@ -15,7 +15,7 @@ class IndexerTest extends BaseTest
 	/**
 	 * @group Slow
 	 * @slowExecutionTime 0.08802 ms
-	 * @covers Wikia\Search\Indexer::getPages
+	 * @covers Wikia\Search\Indexer::getItems
 	 */
 	public function testGetPages() {
 		$indexer = $this->getMockBuilder( 'Wikia\Search\Indexer' )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Wikia\Search\Indexer" 21 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)